File: flutterUI/flutter_mobile/lib/models/vrsta.dart
import 'package:json_annotation/json_annotation.dart';

part 'vrsta.g.dart';

@JsonSerializable()
class Vrsta {
  int? vrstaId;
  String? naziv;
  String? rasa;
  String? opis;
  String? boja;
  int? starost;
  bool? prostor;

  Vrsta({
    this.vrstaId,
    this.naziv,
    this.rasa,
    this.opis,
    this.boja,
    this.starost,
    this.prostor,
  });

  factory Vrsta.fromJson(Map<String, dynamic> json) => _$VrstaFromJson(json);

  Map<String, dynamic> toJson() => _$VrstaToJson(this);
}

File: eKucniLjubimci.Services/Database/Vrsta.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace eKucniLjubimci.Services.Database
{
    public class Vrsta
    {
        public int VrstaId { get; set; }
        public string Naziv { get; set; }
        public string Opis { get; set; }
        public string Boja { get; set; }
        public int Starost { get; set; }
        public bool Prostor { get; set; }
        public bool isDeleted { get; set; } = false;

        public virtual List<Zivotinja> Zivotinje { get; set; }

        public int RasaId { get; set; }
        public virtual Rasa Rasa { get; set; }


    }
}
